src/components/formatSchoolName/formatSchoolName.tsx

```diff
@@ -0,0 +1,33 @@
+import { ReactNode } from "react";
+
+import { Span } from "../Typography";
+
+export const formatSchoolName = (
+  schoolName: ReactNode,
+  inputValue: string | undefined,
+) => {
+  const schoolNameString = `${schoolName}`;
+  const inputValueString = inputValue || "";
+  const regexPattern = new RegExp(inputValueString, "i");
+  const firstIndexOfPattern = schoolNameString.search(regexPattern);
+  const splitSchoolName = schoolNameString.split(regexPattern);
+  const sliceToMakeBold = schoolNameString.slice(
```

Comment:
is the `sliceToMakeBold` the same as the `inputValueString`?

Reply:
input value string is what the user types in; slice the make bold is the thing thats been matched by the inputvaluestring. the reason why you need both is because its not case sensitive for example. inputValueString = "sch" but whats needs to be bold is "Sch"

Reply:
gotcha! thanks